src/tests/api/Filter.test.js

```diff
@@ -0,0 +1,136 @@
+import { applyFilter } from 'src/api';
+import { filters } from './data-mocks/filters';
+
+const makePossibleFeature = (value) => [
+  {
+    type: 'Feature',
+    geometry: {
+      type: 'Point',
+      coordinates: [0, 0]
+    },
+    properties: {
+      column1: value
+    }
+  },
+  {
+    column1: value
+  }
```

Comment:
This 'rawFeature' concept here looks a bit obscure to me, could you please explain what do we want here? when is that kind of 'object' (not really a feature) useful?

Reply:
With the new viewport-feature approach, 'data' could be a regular feature (GeoJSON feature), or a 'properties' GeoJSON object, like this 'rawFeature' :roll_eyes:

Reply:
But that is "another thing", so we can't call it 'feature', can we? 

If we really need that, then I propose:
1. to split it into a different helper, with a good name, without 'mixing' it with the pure one, as the logic is different...
2. to remove it and use an 'inline object' in the required tests